NOTICE.md

```diff
@@ -0,0 +1,10 @@
+## Declared Project Licenses
+
+This program and the accompanying materials are made available under the terms of the Apache License, Version 2.0 which is available at https://www.apache.org/licenses/LICENSE-2.0.
+
+SPDX-License-Identifier: Apache-2.0
+
+## Contributors:
+*   Robert Bosch GmbH - initial API and implementation
+*   Microsoft Corporation - initial API and implementation
+*   Contributors to the Eclipse Foundation - major maintaince
```

Comment:
"Contributors to the Eclipse Foundation" isn't a contributor by itself - it's just the "reference" to this list of contributors. So, actual contributors are just MS and Bosch
```suggestion
```